.companion.rc

```diff
@@ -29,6 +29,7 @@ if [ ! -f /home/pi/.updating ]; then
 	sudo -H -u pi screen -dm -S nmearx $COMPANION_DIR/tools/nmea-receiver.py
 	sudo -H -u pi screen -dm -S wldriver $COMPANION_DIR/tools/underwater-gps.py --ip=waterlinked.local --port=80
 	sudo -H -u pi screen -dm -S bridgemanager $COMPANION_DIR/tools/ping360_bridge_manager.py
+	sudo -H -u pi screen -dm -S mavlink2rest $COMPANION_DIR/tools/mavlink2rest --connect udpin:0.0.0.0:9002 --server 0.0.0.0:4778
```

Comment:
```suggestion
	sudo -H -u pi screen -dm -S mavlink2rest $COMPANION_DIR/tools/mavlink2rest --connect udpin:127.0.0.1:9002 --server 0.0.0.0:4778
```
Can you test it with `127.0.0.1` or `localhost`? It should work while being explicit about where the data is coming from.

Reply:
good catch, it works.